CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,149 @@
+# Code of Conduct
+
+Contact: a moderator ([see below](#moderators)), or a member of the WPT
+community that you feel you can trust.
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
```

Comment:
I feel like this should end with "or any other attribute" or something; the intent is that harassment in general is unacceptable so the list of characteristics is presumably for guidance rather than being exhaustive (this is different from the legal concept of "protected classes" for example where the list of characteristics is exhaustive). For example the [Mozilla CPG](https://www.mozilla.org/en-US/about/governance/policies/participation/) uses "OR any other dimension of diversity" at the end of a similar list.

Reply:
Done, reusing "any other dimension of diversity" (I wasn't comfortable using 'attribute' in this context).